Build/17.0/packages.config

```diff
@@ -84,4 +84,5 @@
   <package id="System.Threading.Tasks.Extensions" version="4.5.4"/>
   <package id="Microsoft.VisualStudio.RpcContracts" version="17.0.50-preview-0002-0005" />
   <package id="Microsoft.DiaSymReader.Pdb2Pdb" version="1.1.0-beta2-21181-01" />
+  <package id="Microsoft.TestPlatform.ObjectModel" version="17.4.0-preview-20221003-03" />
```

Comment:
should we wait for a none preview version with the same changes?

Reply:
This is what's given to me by the test explorer team. I ran the tests locally and haven't found any issues. Are you worried the preview version is unstable in some way?

Reply:
The new APIs present in the above NuGet package should be available in preview builds of VS 2022 v17.4. The package simply makes it so that you can build without having to have a matching version of VS installed in your CI. But when testing locally, everything should work so long as you test against a recent 17.4 preview build.